GoogleSignIn/Sources/GIDSignIn_Private.h

```diff
@@ -14,36 +14,82 @@
  * limitations under the License.
  */
 
+#import <TargetConditionals.h>
+
 #import "GoogleSignIn/Sources/Public/GoogleSignIn/GIDSignIn.h"
 
+#if __has_include(<UIKit/UIKit.h>)
+#import <UIKit/UIKit.h>
+#elif __has_include(<AppKit/AppKit.h>)
+#import <AppKit/AppKit.h>
+#endif
+
 NS_ASSUME_NONNULL_BEGIN
 
 @class GIDGoogleUser;
 @class GIDSignInInternalOptions;
 
-// Represents a completion block that takes a `GIDUserAuth` on success or an error if the operation
-// was unsuccessful.
+/// Represents a completion block that takes a `GIDUserAuth` on success or an error if the operation
+/// was unsuccessful.
 typedef void (^GIDUserAuthCompletion)(GIDUserAuth *_Nullable userAuth, NSError *_Nullable error);
 
 // Private |GIDSignIn| methods that are used internally in this SDK and other Google SDKs.
 @interface GIDSignIn ()
 
-// Redeclare |currentUser| as readwrite for internal use.
+/// Redeclare |currentUser| as readwrite for internal use.
 @property(nonatomic, readwrite, nullable) GIDGoogleUser *currentUser;
 
-// Private initializer for |GIDSignIn|.
+/// Private initializer for |GIDSignIn|.
 - (instancetype)initPrivate;
 
-// Authenticates with extra options.
+/// Authenticates with extra options.
 - (void)signInWithOptions:(GIDSignInInternalOptions *)options;
 
-// Restores a previously authenticated user from the keychain synchronously without refreshing
-// the access token or making a userinfo request. The currentUser.profile will be nil unless
-// the profile data can be extracted from the ID token.
-//
-// @return NO if there is no user restored from the keychain.
+/// Restores a previously authenticated user from the keychain synchronously without refreshing
+/// the access token or making a userinfo request. The currentUser.profile will be nil unless
```

Comment:
Should the second sentence be listed on a new line below the first so that it's in the "Discussion" portion of the documentation (as opposed to the "Summary," which is typically just one sentence).